python/cudf/cudf/tests/test_dataframe.py

```diff
@@ -11100,3 +11100,8 @@ def test_from_records_with_index_no_shallow_copy():
     data = np.array([(1.0, 2), (3.0, 4)], dtype=[("x", "<f8"), ("y", "<i8")])
     df = cudf.DataFrame(data.view(np.recarray), index=idx)
     assert df.index is idx
+
+
+def test_bool_raises():
+    with pytest.raises(TypeError):
```

Comment:
Can we use `assert_exceptions_equal` to validate and compare pandas errors? Same for below.

Reply:
Sure thing. Changed

Reply:
Thanks!